src/simulation/m_global_parameters.fpp

```diff
@@ -670,12 +710,37 @@ contains
             integral(i)%ymax = dflt_real
         end do
 
-        ! GRCBC flags
-        #:for dir in {'x', 'y', 'z'}
-            bc_${dir}$%grcbc_in = .false.
-            bc_${dir}$%grcbc_out = .false.
-            bc_${dir}$%grcbc_vel_out = .false.
-        #:endfor
+        ! Lagrangian subgrid bubble model
+        lag_bubbles = .false.
+        lag_write_bubbles = .false.
+        lag_write_bubble_stats = .false.
+        lag_bubble_model = dflt_int
```

Comment:
i think these should be in a derived type instead of `lag_XYZ`

Reply:
I have added a derived type to wrap up all the user parameters needed for the Lagrange bubbles.

Reply:
Fixed. The derived type variable is called 'lag_params%XYZ'